Geometry/RPCGeometryBuilder/src/RPCGeometryBuilderFromCondDB.cc

```diff
@@ -143,8 +143,8 @@ RPCGeometry* RPCGeometryBuilderFromCondDB::build(const RecoIdealGeometry& rgeo)
         bp = new BoundPlane(gpOfCentre, refSurf.rotation(), bounds);
       }
       else {
-        float cornersLo[3] = {0,}, cornersHi[3] = {0,};
-        float cornersZ[2] = {0,};
+        float cornersLo[3] = {0,0,0}, cornersHi[3] = {0,0,0};
```

Comment:
Hi @jhgoh -for future work, 

float cornersLo[3]={0};

will also do what you want it to.